df-chat-enhance/module.json

```diff
@@ -1,32 +1,57 @@
 {
-	"name": "{{name}}",
-	"title": "{{title}}",
-	"description": "{{description}}",
-	"version": "{{version}}",
+	"name": "df-chat-enhance",
+	"title": "DF Chat Enhancements",
+	"description": "Various enhancements to the chat features.",
+	"version": "2.0.0",
 	"author": "flamewave000#6766",
 	"minimumCoreVersion": "0.7.9",
 	"compatibleCoreVersion": "0.7.9",
 	"dependencies": [
 		{ "name": "_chatcommands" },
 		{ "name": "lib-wrapper" }
 	],
-	"esmodules": "{{sources}}",
-	"styles": "{{css}}",
+	"esmodules": [
+		"src/CONFIG.js",
+		"src/archive.js",
+		"src/df-chat-enhance.js",
+		"src/edit.js",
+		"src/logger.js",
+		"src/privacy.js",
+		"src/archive/DFChatArchive.js",
+		"src/archive/DFChatArchiveManager.js",
+		"src/archive/DFChatArchiveNew.js",
+		"src/archive/DFChatArchiveViewer.js",
+		"src/archive/df-chat-archive.js",
+		"src/logger/DFAdventureLogConfig.js",
+		"src/logger/DFAdventureLogProcessor.js",
+		"src/logger/df-adventure-log.js",
+		"src/logger/libWrapper.d.js",
+		"src/privacy/df-chat-privacy.js",
+		"src/edit/DFChatEditor.js",
+		"src/edit/df-chat-edit.js"
+	],
+	"styles": [
+		"css/adventure-log.css",
+		"css/archive-manager.css",
+		"css/archive-new.css",
+		"css/chat-editor.css",
+		"css/privacy-button.css"
+	],
 	"languages": [
-		{ "lang": "en", "name": "English", "path": "lang/en.json" }
+		{ "lang": "en", "name": "English", "path": "lang/en.json" },
+		{ "lang": "pt-BR", "name": "Português (Brasil)", "path": "lang/pt-BR.json"}
```

Comment:
Please remove all other changes in the `df-chat-enhance/module.json` file except for these 2 lines in the languages part.